lure.go

```diff
@@ -84,16 +84,26 @@ func runMain(config *project.LureConfig, auth vcs.Authentication) {
 		if err != nil {
 			log.Logger.Fatalf("\"Could not generate guid\" %s", err)
 		}
-		localDestination := "/tmp/" + repoGUID.String()
+		localDestination := os.TempDir() + repoGUID.String()
+
+		var provider command.Repository
+		switch projectConfig.Host {
+		case vcs.GitHub:
+			provider = repository.NewGitHub(auth, projectConfig)
+		case vcs.Bitbucket:
+			provider = repository.NewBitbucket(auth, projectConfig)
+		default:
+			// host = nil
+			err = fmt.Errorf("Unknown Host '%s' - must be one of %s, %s", projectConfig.Host, vcs.GitHub, vcs.Bitbucket)
```

Comment:
should we leave at this point?

Reply:
Yes, I don't know why we don't leave when we don't know the VCS either